README.md

```diff
@@ -43,11 +47,19 @@ sudo pip install git+https://github.com/googlefonts/fontbakery.git; # install fo
 ```sh
 sudo add-apt-repository --yes ppa:fontforge/fontforge;
 sudo apt-get update -qq;
-sudo apt-get install python-fontforge ttfautohint swig libicu-dev;
+sudo apt-get install python-fontforge ttfautohint swig libicu-dev git;
```

Comment:
heh! I'd say github users already have git installed... but yeah it doesn't really hurt to explicitly state the dependency here... :-)